pkgs/applications/office/tusk/default.nix

```diff
@@ -0,0 +1,33 @@
+{ appimageTools, fetchurl, lib, gsettings-desktop-schemas, gtk3 }:
+
+let
+  pname = "tusk";
+  version = "v0.23.0";
+in appimageTools.wrapType2 rec {
+  name = "${pname}-${version}";
+  src = fetchurl {
```

Comment:
Please use ```fetchFromGitHub```.

Reply:
Not sure I understand—the AppImage is not in the git repository, only on the release page

Reply:
I'm sorry, my bad :)